src/snap-points.ts

```diff
@@ -285,12 +285,21 @@ export const createSnapshotForNetwork = async (network: Network) => {
         totalPoints,
         positionsAmount: eventsObject[curr].active.length,
         last24HoursPoints: 0,
+        position: 0,
       };
       return acc;
     },
     {}
   );
 
+  const sortedKeys = Object.keys(points).sort(
```

Comment:
We would like to reduce the time of taking snapshot. Things like calculating points and sorting leaderboard can be maintaned by separated script or directly in API endpoint handler.
If it's alot of changes it can stay within single script but in future it should be adjusted